project-base/tests/ShopBundle/Smoke/Http/RouteConfigCustomization.php

```diff
@@ -202,7 +202,7 @@ private function configureAdminRoutes(RouteConfigCustomizer $routeConfigCustomiz
             })
             ->customizeByRouteName('admin_pricinggroup_delete', function (RouteConfig $config) {
                 /** @var \Shopsys\FrameworkBundle\Model\Pricing\Group\PricingGroup $pricingGroup */
-                $pricingGroup = $this->getPersistentReference(PricingGroupDataFixture::PRICING_GROUP_PARTNER_DOMAIN_1);
+                $pricingGroup = $this->getPersistentReference(PricingGroupDataFixture::PRICING_GROUP_PARTNER_DOMAIN . '_1');
```

Comment:
Why is this still used with `_1` and its not getting it by domainId?

Reply:
fixed in 10cec4dfa